### ISSUE TITLE
Pip installable project

### ISSUE THREAD

**Issue by theSage21** [PyJaipur/PyJudge#110]
It would be nice to be able to pip install this project.

**Comment by rishabhKalakoti** [comment]
isn't this one a gfi?

**Comment by theSage21** [comment]
I'm not sure. If someone reads the docs it could be a GFI but people attempting this don't usually read the docs so it might need some hand holding

**Comment by Shivank98** [comment]
I want to work on it.

**Comment by theSage21** [comment]
Yes. This project needs to be pip installable completely. That's all.

**Comment by Shivank98** [comment]
https://pypi.org/project/Pjudge/
*  couldn't take PyJudge as it was already in use.
*  only metafiles that were created with wheel are downloading.

any points that i missed or i keep doing it on my own.

**Comment by theSage21** [comment]
umm, that name conflicts too much with PyJudge. Maybe call it something like hukum/maharaj . Something rajasthani and about judging

about the setup, I recollect needing to specify where the HTML etc was in the setup file. Take a look at [at the project](https://github.com/theSage21/openJudge/blob/master/setup.py#L24)

I can't seem to find the relevant docs. maybe there's a better way of doing this now?

**Comment by rishabhKalakoti** [comment]
Is this one done?
The project does install on `pip install Pjudge`, I just don't know how to use it with pip though.

**Comment by theSage21** [comment]
nope. A few things need to be done for this to be finished:

- [x] pip install
- [ ] pip install should supply the html files as well
- [ ] project is runnable via `python -m <project>`

**Comment by rishabhKalakoti** [comment]
@Shivank98 plz take this one when you are done with exams :)
The project has been registered with your account for pip.

**Comment by Shivank98** [comment]
I need some help with it. I got the point that we need to mention the extra files like HTML ins setup. I haven't look too deeply but [poetry](https://python-poetry.org/) seems to do that work for us( let me know if better ways are available ). what I need help in how we can run the project with `python -m <project>`. I can think of writing some scripts but that doesn't seems to be right way. @theSage21  how you managed this in openjudge?

**Comment by theSage21** [comment]
1. Poetry is awesome!
2. I usually write a `__main__.py` file which gets called when we do `python -m project`.

Could you please open a PR for what you've already done? That way we can have more productive discussions.
